packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -210,6 +219,10 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
                 }
             }
         }
+        // Tag stack as a CodeArtifact in case it wasn't already
```

Comment:
I'm not sure this is required. In a sense that if caller provided stack, as with any other property, it's responsibility of a caller to properly tag all properties. The difference with prepareErrorObject() flow is that we are extracting it from error object or collecting stack, so we know for sure it may contain info that can identify JS bundle.

I'd say it's Ok to have it here, but then obvious question - why only stack? Maybe _message_ property is also present and was extracted somewhere from error object, so it should be tagged to? We do not know for sure, as meaning of these properties might be different (though it's less likely for stack).

It's basically a questing game / attempting to put safeguards for most obvious property names that we can guess intention with high probability.